README.md

```diff
@@ -103,8 +103,8 @@ class Program
     }
 }
 ```
-
-Note that `RecyclableMemoryStreamManager` should be declared once and it will live for the entire process lifetime. It is perfectly fine to use multiple pools if you desire, especially if you want to configure them differently.
+| **_IMPORTANT_** | Note that `RecyclableMemoryStreamManager` should be declared once and it will live for the entire process lifetime. It is perfectly fine to use multiple pools if you desire, especially if you want to configure them differently..|
+|-|:-|
```

Comment:
```suggestion

⚠️ Note that `RecyclableMemoryStreamManager` should be declared once and it will live for the entire process lifetime. It is perfectly fine to use multiple pools if you desire, especially if you want to configure them differently.
```

Reply:
I think this note is too important for a single moji 😢, but it's better than the current description.
What do you think?

Reply:
Up to you. Just remove doubled dot at the end.

Reply:
thx :)
done